CONTRIBUTING.md

```diff
@@ -63,6 +63,17 @@ Try those commands below:
 2. Run unit test: `make unit-test`
 3. All of the above: `make`
 
+### Tips
+Using default Python Package Index (PyPI, https://pypi.org/) maybe pretty slow due to the network connection.
+Developers can choose using a PyPI mirror, search engine can help you find nearest and fast PyPI mirror.
+
+Setting python using a PyPI mirror as default package source is easy. For pip version >=10.0.0 (if not, please using `pip install pip -U` to upgrade pip):
```

Comment:
`pip install -U pip`?

Reply:
It seems `pip install -U pip` looks better than `pip install pip -U`. I will modify it.